app/src/controllers/chefs.ts

```diff
@@ -6,44 +6,85 @@ import { IdentityProvider } from '../components/constants';
 
 import type { JwtPayload } from 'jsonwebtoken';
 import type { NextFunction, Request, Response } from '../interfaces/IExpress';
-import type { ChefsFormConfig, ChefsFormConfigData, ChefsSubmissionFormExport } from '../types';
+import type { ChefsFormConfig, ChefsFormConfigData, ChefsSubmissionForm, ChefsSubmissionFormExport } from '../types';
 
 const controller = {
   getFormExport: async (req: Request, res: Response, next: NextFunction) => {
     try {
       const cfg = config.get('server.chefs.forms') as ChefsFormConfig;
-      let formData = new Array<ChefsSubmissionFormExport>();
+      let exportData = new Array<Partial<ChefsSubmissionForm & { formId: string }>>();
 
-      // Get a JSON export of all form data merged into a single array
+      // Get CHEFS export of all form data, converted to a modified ChefsSubmissionForm type
       await Promise.all(
         Object.values<ChefsFormConfigData>(cfg).map(async (x: ChefsFormConfigData) => {
           const data = await chefsService.getFormExport(x.id);
-          data.forEach((d: ChefsSubmissionFormExport) => (d.form.id = x.id));
-          formData = formData.concat(data);
+
+          data.forEach((d: ChefsSubmissionFormExport) => {
+            const financiallySupportedValues = {
+              financiallySupportedBC: isTruthy(d.isBCHousingSupported),
+              financiallySupportedIndigenous: isTruthy(d.isIndigenousHousingProviderSupported),
+              financiallySupportedNonProfit: isTruthy(d.isNonProfitSupported),
+              financiallySupportedHousingCoop: isTruthy(d.isHousingCooperativeSupported)
+            };
+
+            exportData.push({
+              formId: x.id,
+              submissionId: d.form.submissionId,
+              confirmationId: d.form.confirmationId,
+              contactEmail: d.contactEmail,
+              contactPhoneNumber: d.contactPhoneNumber,
+              contactName: `${d.contactFirstName} ${d.contactLastName}`,
+              financiallySupported: Object.values(financiallySupportedValues).includes(true),
+              ...financiallySupportedValues,
+              intakeStatus: d.form.status,
+              latitude: d.latitude,
+              longitude: d.longitude,
+              naturalDisaster: d.naturalDisasterInd,
+              projectName: d.companyNameRegistered,
+              queuePriority: d.queuePriority,
+              singleFamilyUnits: d.singleFamilyUnits ?? d.multiFamilyUnits,
+              streetAddress: d.streetAddress,
+              submittedAt: d.form.createdAt,
+              submittedBy: d.form.username
+            });
+          });
```

Comment:
Something about this feels unsafe, especially with the confluence of Promise.all, an ignored map, and external context pushing. On a high level, it is safer to structure the logic more like the following instead (js pseudocode):
```js
const exportData = await Promise.all(Object.values.map(x => {
  return (await chefsService.getFormExport(x.id)).map(d => ({
    // The actual resolved contents you're pushing into the array
  }));
}));
```
This is safer as this guarantees order in the destination array (you are not guaranteed execution order in case there's network latency slowing something down when pushing into an array), and is generally more functional in design as you'd be bubbling up the stack and actually leveraging the await for a specific purpose.